src/runtime/security/init.cpp

```diff
@@ -68,5 +68,22 @@ bool init(bool is_server)
     return true;
 }
 
+bool init_for_zookeeper_client()
+{
+    error_s err = run_kinit();
+    if (!err.is_ok()) {
+        derror_f("initialize kerberos failed, with err = {}", err.description());
+        return false;
+    }
+    ddebug("initialize kerberos for zookeeper client succeed");
```

Comment:
```suggestion
    dinfo("initialize kerberos for zookeeper client succeed");
```

Reply:
I think it's ok to use ddebug here

Reply:
I think we can make a plan to reverse info-level and debug-level, it's confused for most developers :)

Reply:
+1